src/ptvsd/_vendored/pydevd/tests_python/test_debugger_json.py

```diff
@@ -3,10 +3,9 @@
 
 from _pydevd_bundle._debug_adapter import pydevd_schema, pydevd_base_schema
 from _pydevd_bundle._debug_adapter.pydevd_base_schema import from_json
-from tests_python.debugger_unittest import IS_JYTHON, REASON_STEP_INTO, REASON_STEP_OVER, \
-    REASON_CAUGHT_EXCEPTION, REASON_STEP_RETURN, IS_APPVEYOR, overrides
-from _pydevd_bundle._debug_adapter.pydevd_schema import ThreadEvent, ModuleEvent, OutputEvent, \
-    ExceptionOptions, Response, StoppedEvent, ContinuedEvent
+from tests_python.debugger_unittest import IS_JYTHON, IS_APPVEYOR, overrides
```

Comment:
Note: I had to rewrite many tests to use only DAP messages (because internally the thread id is still a string and we have a generated thread id when passed through json -- mixing both APIs in the same test is no longer ok because of that).